src/metabase/analytics/prometheus.clj

```diff
@@ -0,0 +1,248 @@
+(ns metabase.analytics.prometheus
+  "Namespace for collection metrics with Prometheus. Will set up a registry and a webserver on startup
+  if [[prometheus-server-port]] is set to a port number. This can only be set in the environment and not though the
```

Comment:
```suggestion
  if MB_PROMETHEUS_SERVER_PORT is set to a port number. This can only be set in the environment and not though the
```

Is it clearer if we specified the env var here instead of our setting?

Reply:
That's a good point. I'll add some more info there.